src/test/java/org/cactoos/map/MapOfTest.java

```diff
@@ -108,4 +108,15 @@ public void createsMapWithFunctions() {
         );
     }
 
+    @Test
+    public void string() {
+        MatcherAssert.assertThat(
+            "Can't convert to string",
+            new MapOf<Integer, Integer>(
+                new MapEntry<>(-1, 0),
+                new MapEntry<>(1, 2)
+            ).toString(),
+            Matchers.equalTo("-1=0, 1=2")
```

Comment:
@SharplEr Shouldn't it be ``{-1=0, 1=2}``?

Reply:
Done